sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/DefaultOfferRequirementProvider.java

```diff
@@ -225,7 +225,9 @@ public OfferRequirement getExistingOfferRequirement(PodInstanceRequirement podIn
                 podInstance.getIndex(),
                 taskRequirements,
                 ExecutorRequirement.create(getExecutor(podInstance, serviceName, targetConfigurationId)),
-                podInstance.getPod().getPlacementRule());
+                //TODO(MB): do not add placement rules to getExistingOfferRequirement
```

Comment:
What does this TODO mean? It seems like the Optional.empty() satisfies this?

Also, can we write a unit test that exercises this?